server/src/main/java/org/opensearch/rest/action/cat/RestIndicesAction.java

```diff
@@ -104,6 +105,13 @@ public class RestIndicesAction extends AbstractListAction {
         "Parameter [master_timeout] is deprecated and will be removed in 3.0. To support inclusive language, please use [cluster_manager_timeout] instead.";
     private static final String DUPLICATE_PARAMETER_ERROR_MESSAGE =
         "Please only use one of the request parameters [master_timeout, cluster_manager_timeout].";
+    private static final IndicesStatsResponse EMPTY_INDICES_STATS_RESPONSE = new IndicesStatsResponse(
```

Comment:
do we need two seperate constants ? Can this be part of IndicesStatsResponse itself ?

Reply:
Introduced a static get method in `IndicesStatsResponse`.